service/metering.go

```diff
@@ -25,11 +31,13 @@ func sendMetering(ctx context.Context, meter dmetering.Meter, userID, apiKeyID,
 
 		Endpoint: endpoint,
 		Metrics: map[string]float64{
-			"egress_bytes":     float64(egressBytes),
-			"written_bytes":    float64(bytesWritten),
-			"read_bytes":       float64(bytesRead),
-			"wasm_input_bytes": float64(inputBytes),
-			"message_count":    1,
```

Comment:
y'avait-il pas déjà une convention d'avoir l'unité à la fin? donc `uncompressed_read_bytes` plutôt que `uncompressed_bytes_read` ?

Reply:
good point, i will make the change